packages/core/src/lib/iserror.js

```diff
@@ -1 +0,0 @@
-module.exports = require('iserror')
```

Comment:
now that we've replicated this code in TS, we can remove the dependency on this library from `packages/core/package.json`